modules/lightning_features/lightning_core/src/BundleEntityStorage.php

```diff
@@ -57,7 +57,7 @@ public static function createInstance(ContainerInterface $container, EntityTypeI
   public function loadMultiple(array $ids = NULL, $check_access = FALSE) {
     if ($check_access) {
       $ids = array_filter(
-        $this->getQuery()->execute(),
+        $ids ?: $this->getQuery()->execute(),
```

Comment:
@phenaproxima , @balsama -

As mentioned [here](https://github.com/acquia/lightning/pull/370#discussion_r118573023), the specified `$ids` were _always_ being overwritten when `$check_access` was `TRUE`.